packages/material-ui/src/Select/SelectInput.js

```diff
@@ -8,35 +8,33 @@ import { refType } from '@material-ui/utils';
 import ownerDocument from '../utils/ownerDocument';
 import capitalize from '../utils/capitalize';
 import Menu from '../Menu/Menu';
-import { nativeSelectRootStyles, nativeSelectIconStyles } from '../NativeSelect/NativeSelectInput';
+import {
+  nativeSelectSelectStyles,
+  nativeSelectIconStyles,
+} from '../NativeSelect/NativeSelectInput';
 import { isFilled } from '../InputBase/utils';
 import experimentalStyled, { slotShouldForwardProp } from '../styles/experimentalStyled';
 import useForkRef from '../utils/useForkRef';
 import useControlled from '../utils/useControlled';
-import selectClasses, { getSelectUtilityClasses } from './selectClasses';
+import { getSelectUtilityClasses } from './selectClasses';
 
-const SelectRoot = experimentalStyled(
+const SelectSelect = experimentalStyled(
   'div',
   {},
   {
     name: 'MuiSelect',
-    slot: 'Root',
+    slot: 'Select',
     overridesResolver: (props, styles) => {
       const { styleProps } = props;
       return {
-        [`&.${selectClasses.select}`]: {
```

Comment:
We could add a comment on why we bump the specificity, from what I remember, it's important to guarantee that the customization is applied. The base style comes from the InputBase-input which we have no injection order guarantee with.

Ah yes, we say "// Win specificity over the input base" below

Reply:
```suggestion
        // Win specificity over the input base
        '&': {
```

Reply:
@m4theushw what do you think about this comment?

Reply:
Updated. Missed this one because there's already a comment below.